tools/run-mypy

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python2.7
+from __future__ import print_function
+from __future__ import absolute_import
+import os
+import re
+import sys
+import optparse
+import subprocess
+import traceback
+
+import lister
+
+
+targets = """
+zproject
+bots
+zilencer
+analytics
+corporate
+confirmation
+tools
+scripts/setup
+scripts/lib
+*.py
+api
+""".split()
+
+
+# exclude commands
+exclude = """
+test
+fenced_code
+settings
+""".split()
+
+by_lang = lister.list_files(targets=targets, modified_only=False, use_shebang=True,
+    ftypes=['py'], group_by_ftype=True, exclude=exclude, exclude_flag=True)
+
+# mypy command to run silently
+result = subprocess.Popen(["mypy", "--silent", "--py2", "--check-untyped-defs"] + by_lang['py'],
+    stdout = subprocess.PIPE,
+    stderr = subprocess.PIPE)
+for pipe in (result.stdout, result.stderr):
+   for ln in pipe:
+      print(ln)
```

Comment:
This needs to exit with status 1 when it produces output or it will always succeed; currently your `mypy` test is passing even though it produces a bunch of output.